L1Trigger/L1TCaloLayer1/python/simCaloStage2Layer1Digis_cfi.py

```diff
@@ -18,5 +18,5 @@
     unpackEcalMask = cms.bool(False),
     unpackHcalMask = cms.bool(False),
     # See UCTLayer1.hh for firmware version
-    firmwareVersion = cms.int32(1),
+    firmwareVersion = cms.int32(3),
```

Comment:
hi @nsmith-interesting- remind me how this code gets used in production workflows?